prac_08/unreliable_car.py

```diff
@@ -0,0 +1,20 @@
+from prac_08.car import Car
+import random
+
+
+class UnreliableCar(Car):
+
+    def __init__(self, name, fuel, reliability):
+        """Initialise an UnreliableCar instance, based on parent class Car."""
+        super().__init__(name, fuel)
+        self.reliability = reliability
+
+    def drive(self, distance):
+        reliability_check = random.randint(0, 101)
+        print (reliability_check)
```

Comment:
This print should be removed or commented out.
The values here for randint mean it's not 1-100.